plugins/balloonpanel/plugin.js

```diff
@@ -363,6 +408,34 @@
 			};
 
 			return function( element, options ) {
+				if ( element instanceof CKEDITOR.dom.selection ) {
```

Comment:
It's no longer `element`, it's `elementOrSelection`. Subtle change, but it will increase the readability of the rest of the code (`element.isFake` vs `elementOrSelection.isFake`).